backstop/tests/select.html

```diff
@@ -24,10 +24,11 @@
     />
     <style>
       section[id^='demo'] {
-        width: 45vw;
+        width: auto;
```

Comment:
hmm...
Was checking why its needed at all, seems we have different grid implementations.
All select examples have 1fr (coming from `iui-input-container`.
While the example which uses only `.iui-input-with-icon` does not have any rules on width... 
This is expected?
@gretanausedaite @mayank99

Reply:
The `1fr` doesn't automatically give it width. It makes sense that the `input-with-icon` don't have `grid-template-columns` because it is only being used for placing the icon.

The difference only becomes apparent when you specify `fit-content` on parent. Without that, they behave identically.

Reply:
The parent in this PR has `width: auto` defined in the `<head>`.  @veekeys if you pull this locally and build, be sure to refresh the page because for whatever reason it first renders as if `width: fit-content` is set (at least on my side).

Reply:
Update:  I swapped out the Lorem ipsum in the last example and it is shrinking where I thought it would stay full width.
![Screen Shot 2022-02-22 at 3 45 05 PM](https://user-images.githubusercontent.com/849817/155216210-64b3a732-b0c8-45b9-9b6f-29ef07ed0f00.png)

Reply:
And where is that `fit-content` set?

Reply:
1. `width: fit-content` is set in demo.css
2. `width: auto` is applied on all sections on this page
3. This "in context" demo is using the `<section>` as the flex container, that's why it gets messed up.

I suggest adding a wrapper inside this section to avoid fighting with previous rules.
![image](https://user-images.githubusercontent.com/9084735/155370324-c46b6413-ca4d-40fa-b38e-336f3790ac56.png)

Reply:
My point was to figure, why selects are acting different, not to set custom rules on container to fix that...
but I guess doesnt matter :/